addon/components/listbox.js

```diff
@@ -57,6 +57,7 @@ export default class ListboxComponent extends Component {
   @action
   handleButtonClick(e) {
     if (e.button !== 0) return;
+    e.preventDefault();
```

Comment:
This change is not specific to page-scroll, but related: 

This fixes a bug when the listbox is embedded in a `<form>`. This prevents the form from being submitted when the trigger button is clicked.